contract/test/test-vote-by-committee.js

```diff
@@ -15,10 +15,22 @@ import { mockWalletFactory, seatLike } from './wallet-tools.js';
 import { INVITATION_MAKERS_DESC } from '../src/platform-goals/start-governed-contract.js';
 import { installGovContracts } from './lib-gov-test/puppet-gov.js';
 
-/** @typedef {import('./wallet-tools.js').MockWallet} MockWallet */
+/**
+ * @import {Brand} from '@agoric/ertp/src/types.js';
+ * @import {QuestionDetails} from '@agoric/governance/src/types.js';
+ * @import {OfferSpec} from '@agoric/smart-wallet/src/offers.js';
+ * @import {MockWallet} from './wallet-tools.js';
+ * @import {ExecutionContext, TestFn} from 'ava';
+ * @import {ParamChangesOfferArgs} from '@agoric/inter-protocol/src/econCommitteeCharter.js';
+ */
+
+
```

Comment:
please address all the lint warnings

Reply:
yeah, this slipped in unintentionally. :/